projects/components/src/summary-value/summary-value.component.ts

```diff
@@ -28,6 +28,9 @@ export class SummaryValueComponent implements OnChanges {
   @Input()
   public tooltip?: string;
 
+  @Input()
+  public summaryValueDisplayStyle: SummaryValueDisplayStyle = SummaryValueDisplayStyle.Text;
```

Comment:
Probably this could have been avoided if we would just put summary-value inside an ht-link(as commented in the other PR)

```
<ht-link [params]="<>">
 <ht-summary-value></ht-summary-value>
 </ht-link>
```

Reply:
We may have to update the color to currentColor. I think I remember using this approach somewhere.

Reply:
If that works, that's better and let's revert this